src/main.rs

```diff
@@ -54,10 +54,10 @@ fn _main() -> Result<()> {
         release: matches.is_present("release"),
     };
 
-    let depth_str = matches.value_of("depth").expect("'depth' should be exists");
+    let depth_str = matches.value_of("depth").expect("please provide a depth");
```

Comment:
The variable 'depth' has default value. So, this message is not for user.
I'll revert this after merge.